app-dicts/aspell-be/aspell-be-0.01-r1.ebuild

```diff
@@ -1,20 +1,21 @@
 # Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
+EAPI=6
+
 ASPELL_LANG="Belarusian"
 
-inherit eutils aspell-dict
+inherit aspell-dict-r1
 
-LICENSE="GPL-2"
+SRC_URI="mirror://gnu/aspell/dict/be/aspell5-be-${PV}.tar.bz2"
 
+LICENSE="GPL-2"
 KEYWORDS="alpha amd64 arm ~arm64 hppa ia64 m68k ~mips ppc ppc64 s390 sh sparc x86 ~amd64-fbsd ~sparc-fbsd ~x86-fbsd"
-
-SRC_URI="mirror://gnu/aspell/dict/be/aspell5-be-${PV}.tar.bz2"
 IUSE="classic"
 
 S="${WORKDIR}/aspell5-be-${PV}"
 
-src_unpack() {
-	unpack ${A}
-	use classic || epatch "${FILESDIR}"/aspell5-be-${PV}-official.patch
+src_prepare() {
+	! use classic && PATCHES+=( "${FILESDIR}"/aspell5-be-${PV}-official.patch )
```

Comment:
Seriously? Use `||`. Also where are PATCHES defined that you're appending to them?